src/plugins/paginate/paginate-en.hbs

```diff
@@ -25,15 +25,16 @@
 
 <h2>On this page:</h2>
 <ul>
-	<li><a href="#paginatedList">Example 1 - List</a></li>
+	<li><a href="#paginatedList">Example 1 - List with buttons</a></li>
```

Comment:
Could you please move the example to that last instead of being the first. We would like users to focus on other pagination parameters before this one (same for FR).

Reply:
Fixed in commit "Made recommended changes".